trunner/target/armv7a7.py

```diff
@@ -0,0 +1,112 @@
+from typing import Callable, Optional
+
+from trunner.dut import Dut, SerialDut
+from trunner.harness import (
+    HarnessBuilder,
+    PloInterface,
+    PloImageLoader,
+    PloImageProperty,
+    ShellHarness,
+    Rebooter,
+    RebooterHarness,
+)
+
+from trunner.harness import TerminalHarness
+from trunner.host import Host
+from trunner.tools import Phoenixd, Psu
+from trunner.types import TestResult, TestOptions
+from .base import TargetBase, find_port
+
+
+class ARMv7A7TargetRebooter(Rebooter):
+    def _set_flash_mode(self, flash):
+        self.host.set_flash_mode(not flash)
+
+
+class PsuPloLoader(TerminalHarness, PloInterface):
+    def __init__(self, dut: Dut, psu: Psu):
+        TerminalHarness.__init__(self)
+        PloInterface.__init__(self, dut)
+        self.psu = psu
+
+    def __call__(self):
+        """Loads plo image to RAM using psu tool and erases an area intended for rootfs."""
+        self.psu.run()
+        self.wait_prompt()
+        self.cleanmarkers("2.0", True, ("0x10", "0x1f0", "0x10000", "0x10"), 90)
+
+
+class ARMv7a7Target(TargetBase, PloInterface, Rebooter):
+    image = PloImageProperty(file="phoenix.disk", source="usb0", memory_bank="nor0")
+    plo_psu_script = "plo-ram.sdp"
+    rootfs = False
```

Comment:
Why False? This target has got a rootfs.

```suggestion
    rootfs = True
```